package.json

```diff
@@ -606,6 +606,12 @@
                         "markdownDescription": "Experimental: Prevents you from entering unmatched closing brackets when in `strict` mode. (Does not work when there is an active selection.)",
                         "default": false,
                         "scope": "window"
+                    },
+                    "calva.paredit.killAlsoCutsToKeyboard": {
+                        "type": "boolean",
+                        "markdownDescription": "When enabled, copies the deleted code to the clipboard simulating the Emacs implementation.",
```

Comment:
Emacs users might take this as if there is a kill-ring they can add to. Maybe add some note about that consecutive kills are not adding to the clipboard, but instead replace the clipboard content?

Reply:
Updated. No longer references Emacs so should be less confusing to Emacs users.